src/citrine/_serialization/properties.py

```diff
@@ -248,8 +248,6 @@ def _deserialize(self, value: str) -> uuid.UUID:
         return uuid.UUID(value)
 
     def _serialize(self, value: uuid.UUID) -> str:
-        if not issubclass(type(value), uuid.UUID):
-            raise ValueError('Value {} is not a UUID'.format(value))
```

Comment:
these lines are logically unreachable because the super().serialize() method already checks for the proper underlying type